fbw-a380x/src/wasm/fbw_a380/build.sh

```diff
@@ -9,7 +9,7 @@ if [ "$1" == "--debug" ]; then
   WASMLD_ARGS=""
   CLANG_ARGS="-g"
 else
-  WASMLD_ARGS="-O2 --lto-O2 --strip-debug"
+  WASMLD_ARGS="-O2 --lto-O2 --strip-debug --gc-sections"
```

Comment:
As above, keep common.
```suggestion
  WASMLD_ARGS="-O2 --lto-O2 --strip-debug"
```